src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -26,6 +26,43 @@
 
 namespace Microsoft.ML.Transforms
 {
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="NormalizingTransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | <xref:System.Single> or<xref:System.Double> or a known-sized vector of those types. |
+    /// | Output column data type | The same data type as the input column |
+    ///
+    /// The resulting NormalizingEstimator will normalize the data in one of the following ways based upon how it was created:
+    /// * Min Max - A linear rescale that is based upon the minimum and maximum values for each row.
+    /// * Mean Variance - Rescale each row to unit variance and, optionally, zero mean.
+    /// * Log Mean Variance - Rescale each row to unit variance based on a log scale.
+    /// * Binning - Bucketizes the data in each row and performs a linear rescale based on the calculated bins.
```

Comment:
The explanation of binning doesn't carry much information. Also, we need equations for all of them.

Reply:
Talked offline and agreed to address later. Having equations for all of our transformer/trainers is a much larger issue to address.

---
In reply to: [277110413](https://github.com/dotnet/machinelearning/pull/3432#discussion_r277110413) [](ancestors = 277110413)